src/main/java/at/hannibal2/skyhanni/features/fishing/SeaCreatureFeatures.kt

```diff
@@ -79,6 +80,10 @@ object SeaCreatureFeatures {
     @HandleEvent
     fun onMobDespawn(event: MobEvent.DeSpawn.SkyblockMob) {
         rareSeaCreatures.remove(event.mob)
+
+        val creature = SeaCreatureManager.allFishingMobs[event.mob.name] ?: return
```

Comment:
the mob desapawn even is not entirely correct. we only wanna handle deaths, not despawns bc of 6 min mark reached. 
maybe use EntityDeathEvent or EntityHealthUpdateEvent